<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roboAxes</class>
 <widget class="QDialog" name="roboAxes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>469</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>机器人</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Joint</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="9" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Wrist</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="sBoxBase">
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Base</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Big Arm</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QDoubleSpinBox" name="sBoxBigArm">
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Little Arm</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QDoubleSpinBox" name="sBoxLitArm">
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QDoubleSpinBox" name="sBoxWrist">
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QDoubleSpinBox" name="sBoxHand">
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="5">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnZeroLitArm">
        <property name="text">
         <string>To Default</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnZeroBigArm">
        <property name="text">
         <string>To Default</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnZeroBase">
        <property name="text">
         <string>To Default</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QSlider" name="sldBase">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QSlider" name="sldBigArm">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="QSlider" name="sldLitArm">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btnZeroWrist">
        <property name="text">
         <string>To Default</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="4">
       <widget class="QSlider" name="sldWrist">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QPushButton" name="btnZeroWrist_2">
        <property name="text">
         <string>To Default</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="4">
       <widget class="QSlider" name="sldHand">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Robot</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>To Default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sBoxBase</tabstop>
  <tabstop>sBoxBigArm</tabstop>
  <tabstop>sBoxLitArm</tabstop>
  <tabstop>sBoxWrist</tabstop>
  <tabstop>sBoxHand</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>btnZeroBase</tabstop>
  <tabstop>sldBase</tabstop>
  <tabstop>btnZeroBigArm</tabstop>
  <tabstop>sldBigArm</tabstop>
  <tabstop>btnZeroLitArm</tabstop>
  <tabstop>sldLitArm</tabstop>
  <tabstop>btnZeroWrist</tabstop>
  <tabstop>sldWrist</tabstop>
  <tabstop>btnZeroWrist_2</tabstop>
  <tabstop>sldHand</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
